### ISSUE TITLE
Capes get unequipped after changing a skin until reconnect

### ISSUE THREAD

**Issue by Ryhon0** [IMB11/SkinShuffle#136]
### Describe the issue

If you change your skin on a server with the mod or plugin installed, your cape will become unequipped.
As I understand, this is because the skin signing is done by Mineskin and they can't support capes.
A possible fix would involve ditching Mineskin and signing the skins locally. Mineskin just uploads the skin to some account and keeps checking the skin API response until the skin URL matches.
I'm working on a mod that lets you change capes without reconnecting similar to this one since cape support isn't planned

### SkinShuffle version

2.4.0+1.21.3

**Comment by IMB11** [comment]
I'll have to fix this for 2.6.0, literally just published 2.5.0 🤭 - didn't realise Mineskin reset capes.

**Comment by Ryhon0** [comment]
From the Mineskin source code:
https://github.com/MineSkin/api.mineskin.org/blob/master/src/generator/Generator.ts#L1024
* Skin is uploaded to an account
* URL is generated based on the hash of the file
* Keep trying the skin API till the URL matches

**Comment by IMB11** [comment]
Yeah that isn't possible locally unfortunately, I will look into the capes being unequipped issue though.

**Comment by IMB11** [comment]
I was unable to find a fix for 2.6.0, I will investigate further at a later date, probably after Christmas now im pretty burned out.

**Comment by IMB11** [comment]
Hi, this unfortunately isn't possible due to MineSkin limitations, I can look into making a client-side fix for this though. Will re-open once mineskin adds support for cape signing,
